marginalia.el

```diff
@@ -730,16 +762,47 @@ These annotations are skipped for remote paths."
       (marginalia--fields ("*Remote*" :face 'marginalia-documentation))
     (when-let (attributes (file-attributes (substitute-in-file-name (marginalia--full-candidate cand)) 'string))
```

Comment:
Please change this from 'string to 'integer.

Reply:
For some reason, I've been going through your comments backwards. Please disregard my later comments on "int or string", as what you intend is now clear to me.

Reply:
Haha, okay I see.

Reply:
We're on the same page now at least :stuck_out_tongue:. I've got a hopefully-more-intelligent question: do you know how I should go from UID/GID to user/group name for displaying?

Reply:
There are some functions like user-login-name.

Reply:
I think I've got it now :+1: